onyo/lib/onyo.py

```diff
@@ -326,6 +351,15 @@ def add(self, targets: Union[Iterable[Union[Path, str]], Path, str]) -> None:
                 raise FileNotFoundError(f"'{t}' does not exist.")
 
         self._git(['add'] + tgts)
+        # `Repo.add()` is used by most repo-changing commands, and it might be
+        # used often by the API to change the repository, before manually
+        # calling clear_cache() or after a file was changed with a subprocess
+        # call. To always secure the integrity of the caches, we reset them all.
+        self.clear_caches(assets=True,  # revert e.g. `onyo set`
```

Comment:
Copy paste error; the comments all reference "revert"

Reply:
It seems to be a vocabulary error, I changed them all from reverse. I will look up the different words and check again for which situations they are appropriate 🙈